{"refactored_code/one_shot/public-apis/scripts/validate/format.py": {"error": "expected an indented block after 'for' statement on line 19 (<unknown>, line 20)"}, "refactored_code/one_shot/public-apis/scripts/validate/__init__.py": {"mi": 100.0, "rank": "A"}, "refactored_code/one_shot/public-apis/scripts/validate/links.py": {"mi": 52.8737908583401, "rank": "A"}, "refactored_code/one_shot/public-apis/tests/test_format.py": {"mi": 43.979058441869775, "rank": "A"}, "refactored_code/one_shot/public-apis/tests/test___init__.py": {"mi": 73.78610592223512, "rank": "A"}, "refactored_code/one_shot/public-apis/tests/test_links.py": {"mi": 38.76398890881956, "rank": "A"}}
